packages/core/src/__tests__/initialize.test.ts

```diff
@@ -134,6 +144,27 @@ describe("Android: initialize", () => {
     generatedGlobalErrorFunc(Error("Test"));
     expect(previousHandler).toHaveBeenCalled();
   });
+
+  test("applying previousHandler and throwing a component error", async () => {
+    const previousHandler = jest.fn();
+    ErrorUtils.getGlobalHandler = previousHandler;
+    const result = await initialize({patch: testValue});
+    expect(result).toBe(true);
+
+    const generatedGlobalErrorFunc = handleGlobalError(
+      previousHandler,
+      logIfComponentError,
+    );
+    const componentError = new Error("Test") as ComponentError;
+    componentError.componentStack = "in SomeScreen/n in SomeOtherScreen";
```

Comment:
should include the "at undefined" stuff in the input and then verify it is removed in the output